src/test/java/Homework19.java

```diff
@@ -0,0 +1,44 @@
+import com.beust.jcommander.Parameter;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework19 extends BaseTest{
+
+    @Test
+    public void deletePlaylist() throws InterruptedException {
+        String deletedPlaylistSuccessMessage = "Deleted playlist \"Oreo.\"";
+        provideEmail("kaflimeerim@gmail.com");
+        providePassword("te$t$tudent");
+        loginToKoel();
+        selectPlaylist();
+        //Thread.sleep(2000);
+        clickDeletePlaylistBtn();
+        //Thread.sleep(2000);
+        Assert.assertEquals(getPlaylistDeletedMessage(),deletedPlaylistSuccessMessage);
+       // Assert.assertTrue(wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("div.success.show"))).isDisplayed();
+
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
   
   For example

![Clean_Test](https://github.com/Test-Pro-Bootcamp/A55-QA-Automation-Framework/assets/104761482/ff272844-47a5-4c03-b492-de515ea6b899)